infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerRuntimeInfrastructure.java

```diff
@@ -80,7 +80,10 @@ protected RuntimeContext internalPrepare(
 
   @Override
   public Set<RuntimeIdentity> getIdentities() throws InfrastructureException {
-    return containers.findIdentities();
+    // Due to https://github.com/eclipse/che/issues/5814, recovering is not fully possible
+    // so infrastructure should not claim support of it.
+    // return containers.findIdentities();
+    throw new UnsupportedOperationException("Runtimes tracking currently does not supported.");
```

Comment:
"Runtimes tracking currently does not supported." should be "Runtimes tracking is not supported."